site2/docs/admin-api-overview.md

```diff
@@ -88,3 +83,8 @@ PulsarAdmin admin = PulsarAdmin.builder()
 .build();
 ```
 <!--END_DOCUSAURUS_CODE_TABS-->
+
+## How to define Pulsar resource names when running Pulsar in Kubernetes
+If you run Pulsar Functions or connectors on Kubernetes, you need to follow Kubernetes naming convention to define your Pulsar resource name, whichever admin interface you use.
```

Comment:
I think it's not just the name of Pulsar functions or connectors, others Pulsar components also have the same issue.

Reply:
You're right, that's why we say when you define your Pulsar resource names, you need to follow K8S naming convention (only when you use K8S runtime). To make it more clear, maybe it's better to say "...define the names of your Pulsar resources..."
However, if you don't use K8S runtime, you don't need to follow K8S convention. Pulsar supports more characters than K8S naming convention.